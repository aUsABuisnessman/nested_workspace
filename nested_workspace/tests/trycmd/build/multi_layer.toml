args = ["build", "-vv"]

[bin]
name = "cargo"

[fs]
cwd = "../../../../fixtures/multi_layer"
