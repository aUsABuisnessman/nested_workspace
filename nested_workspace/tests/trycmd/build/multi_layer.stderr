...
   Compiling multi_layer v0.1.0 ([CWD])
...
[multi_layer 0.1.0] <<< [CWD]/workspace
...
[multi_layer 0.1.0]    Compiling single_layer v0.1.0 ([CWD]/workspace/single_layer)
...
[multi_layer 0.1.0] [single_layer 0.1.0] <<< [CWD]/workspace/single_layer/workspace
...
[multi_layer 0.1.0] [single_layer 0.1.0]    Compiling package v0.1.0 ([CWD]/workspace/single_layer/workspace/package)
...
[multi_layer 0.1.0] [single_layer 0.1.0]     Finished `dev` profile [unoptimized + debuginfo] target(s) in [..]s
[multi_layer 0.1.0] [single_layer 0.1.0] >>> [CWD]/workspace/single_layer/workspace
...
[multi_layer 0.1.0]     Finished `dev` profile [unoptimized + debuginfo] target(s) in [..]s
[multi_layer 0.1.0] >>> [CWD]/workspace
...
    Finished `dev` profile [unoptimized + debuginfo] target(s) in [..]s
